src/args.jl

```diff
@@ -1491,6 +1489,10 @@ function _update_axis(axis::Axis, plotattributes_in::AKW, letter::Symbol, subplo
         # then get those args that were passed with a leading letter: `xlabel = "X"`
         lk = Symbol(letter, k)
         if haskey(plotattributes_in, lk)
+            # warn against using range in x,y,z lims
+            if k==:lims && plotattributes_in[lk] isa AbstractRange
```

Comment:
Does this also warn for `plot(0:1, 0:1, lims = -2:2)`?

Reply:
it does, which is weird because I thought in that case `lk` won't be in the keys

Reply:
Ok, wouldn't have expected that, but great!